### PR TITLE
install: write device name and a Wi-Fi credential in image

### PR DESCRIPTION
**Pull request by seiyanuta**

### CODE REVIEW

File: install.go [hunk 1]
```diff
@@ -29,23 +37,52 @@ var installCommand = cli.Command {
...
+		},
 	},
 }
 
-func doInstall(c *cli.Context) error {
-	board := c.String("board")
-	serial := c.String("serial")
-	image := c.String("image")
+func replaceAndFillBytes(content []byte, old, new string) ([]byte, error) {
+	old_bytes := []byte(old)
```
Comment (sh19910711) [inline install.go]: s/old_bytes/oldBytes/

File: install.go [hunk 2]
```diff
@@ -73,5 +110,30 @@ func doInstall(c *cli.Context) error {
 		}
 	}
 
-	return installer(serial, image)
+	// Copy the original image to a temporary file.
+	tmp_image, err := ioutil.TempFile("", "codestand")
```
Comment (sh19910711) [inline install.go]: ditto

Comment (seiyanuta) [inline install.go]: fixed

**Comment by sh19910711** [comment]
Thanks :+1:

**Pull request merged**